manifests/p/Prometheus/WindowsExporter/0.30.0/Prometheus.WindowsExporter.installer.yaml

```
@@ -0,0 +1,48 @@
+# Created with komac v2.9.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: Prometheus.WindowsExporter
+PackageVersion: 0.30.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ReleaseDate: 2025-01-14
+Installers:
+- Architecture: x64
+  InstallerType: portable
+  InstallerUrl: https://github.com/prometheus-community/windows_exporter/releases/download/v0.30.0/windows_exporter-0.30.0-amd64.exe
+  InstallerSha256: 95299387198FFF9514F1B519C1ED847ABF5EE3823AF427CD4CDFAA936F930735
+- Architecture: arm64
+  InstallerType: portable
+  InstallerUrl: https://github.com/prometheus-community/windows_exporter/releases/download/v0.30.0/windows_exporter-0.30.0-arm64.exe
+  InstallerSha256: 29DA5295C98757C762426F270BBD5A141A0B07234AC4A97D5DA0871B52A8C255
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: wix
+  Scope: machine
+  InstallerUrl: https://github.com/prometheus-community/windows_exporter/releases/download/v0.30.0/windows_exporter-0.30.0-amd64.msi
+  InstallerSha256: 21290ED6EEEF383483BA8C20BB1EBB61A843D80120EA9DFAE67B28A614FE7F24
+  ProductCode: '{A7B06148-F35F-485E-A77E-5787916498D8}'
+  AppsAndFeaturesEntries:
+  - Publisher: prometheus-community
+    ProductCode: '{A7B06148-F35F-485E-A77E-5787916498D8}'
+    UpgradeCode: '{66A6EB5B-1FC2-4B14-A362-5CEEC6413308}'
+  InstallationMetadata:
+    DefaultInstallLocation: '%ProgramFiles%\windows_exporter\textfile_inputs'
```

```suggestion
    DefaultInstallLocation: '%ProgramFiles%\windows_exporter\'
```

Not sure what this exactly means, but the binary is placed here. and `textfile_inputs` is just an directory for system admins.